types/test.ts

```diff
@@ -192,4 +192,43 @@ switch(allthings.type) {
     break;
   default:
     break;
-}
\ No newline at end of file
+}
+
+// Test exception constructor with just message.
+try {
+  const exp = new Handlebars.Exception('Just a test message.');
```

Comment:
Please use longer descriptive names, no shorthands (`exception` rather than `exp`). This is also not yet done everywhere, but I will try to have an eye on such things from now on.